plugins/headers.rst

```diff
@@ -0,0 +1,143 @@
+Header Plugins
+==============
+
+Header plugins are useful to manage request headers. Many operations are possible:
+
+Default headers values
+----------------------
+
+The plugin Http\Plugins\HeaderDefaultPlugin allows to set default values for given headers.
+That means if the request hasn't already a given header it will be set,
```

Comment:
That means if a header is not set on a request, it will be added. However, if the header already is present on the request, the request is left unchanged.